cmd/queue/main.go

```diff
@@ -72,13 +72,14 @@ var (
 )
 
 type config struct {
-	ContainerConcurrency     int    `split_words:"true" required:"true"`
-	QueueServingPort         string `split_words:"true" required:"true"`
-	UserPort                 string `split_words:"true" required:"true"`
-	RevisionTimeoutSeconds   int    `split_words:"true" required:"true"`
-	ServingReadinessProbe    string `split_words:"true" required:"true"`
-	EnableProfiling          bool   `split_words:"true"` // optional
-	EnableHTTP2AutoDetection bool   `split_words:"true"` // optional
+	ContainerConcurrency       int    `split_words:"true" required:"true"`
+	QueueServingPort           string `split_words:"true" required:"true"`
+	UserPort                   string `split_words:"true" required:"true"`
+	RevisionTimeoutSeconds     int    `split_words:"true" required:"true"`
+	ServingReadinessProbe      string `split_words:"true" required:"true"`
+	RevisionIdleTimeoutSeconds int    `split_words:"true" default:"-1"` // optional
```

Comment:
Nit: I'd send this as a separate PR where this is actually getting hooked up and pass the canonical "disabling" value directly below for now.

Is there a reason for picking a negative value over 0 as "disabled"?